static-instrumenter/agent-instrumenter/src/test/java/io/opentelemetry/contrib/statical/instrumenter/JarTestUtil.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.statical.instrumenter;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
```

Comment:
Use assertj

Reply:
Frankly I don't see a benefit of assertJ here as assertions are trivial.

Reply:
It's to be consistent in our codebase, we always use assertj regardless of the complexity of the assertions

Reply:
OK, done.